snapcraft/cli/_options.py

```diff
@@ -73,9 +73,10 @@ def __repr__(self):
 _PROVIDER_OPTIONS: List[Dict[str, Any]] = [
     dict(
         param_decls="--target-arch",
+        envvar="SNAPCRAFT_TARGET_ARCH",
         metavar="<arch>",
         help="Target architecture to cross compile to",
-        supported_providers=["host", "lxd", "multipass"],
+        supported_providers=["host", "lxd", "managed-host", "multipass"],
```

Comment:
this is not described in the PR ;-)

might also require its own PR to assess whether we want this for the older functionality